CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## Changed
+- **Breaking API Change**: Restricted type of `BitcoinWallet`'s `network` parameter to improve type safety.
+- **Breaking API Change**: Replaced `InMemoryBitcoinWallet` with `BitcoindWallet`.
```

Comment:
Both of these don't have to be breaking changes.

I would consider the first one to be a bugfix and the latter can be done by just deprecating `InMemoryBitcoinWallet` first and only adding the other.

Just makes for a smoother process overall because the dependency updates can happen automatically and we can just seamlessly transition :)

You don't have to change it now but I would encourage you (@bonomat / @luckysori, donno who actually wrote/decided this) to try and do things in a non-breaking way the next time :)
At the moment we are our own users so it just makes our life easier.

Reply:
Yep, you're right. I considered doing so, but I was too keen on removing bcoin. I've reverted the changes now and deprecated bcoin instead :D